certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1739,6 +1746,30 @@ public class Script : ScriptBase
       response.Content = new StringContent(newBody.ToString(), Encoding.UTF8, "application/json");
     }
 
+    if ("ListEnvelopeDocuments".Equals(this.Context.OperationId, StringComparison.OrdinalIgnoreCase))
+    {
+      var body = ParseContentAsJObject(await response.Content.ReadAsStringAsync().ConfigureAwait(false), false);
+      var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
+      JArray envelopeDocuments = new JArray();
+      JObject newBody = new JObject();
+
+      if (body["envelopeDocuments"] == null)
+      {
+        throw new ConnectorException(HttpStatusCode.BadRequest, "ValidationFailure: Envelope do not contain documents");
+      }
+
+      foreach(var document in body["envelopeDocuments"] as JArray ?? new JArray())
+      {
+        envelopeDocuments.Add(new JObject(
```

Comment:
The shorthand is working only when JObject has a reference defined like this:
var workflowObj = new JObject()
        {
          ["type"] = id["workflowId"],
          ["name"] = id["defaultName"]
        };
       I will leave the code the way it is. If I give a reference name I have to change the schema.

Reply:
If you do something like the following, you don't need to change any schema and still use the shorthand.
```javascript
envelopeDocuments.Add(new JObject()
{
   ["documentId"] = document["documentId"],
    ["name"] = document["name"]
}
```

Reply:
That worked great! Thank you.